### PR TITLE
Template for support subdomains: Fourthwall

### PR DESCRIPTION
**Pull request by beholders-eye**
Dropping #206 in favor of this one

There's no need to update `goentri.com.fourthwall.json`, just need to add a `.subdomain.json` instead.

### CODE REVIEW

File: goentri.com.fourthwall.subdomain.json [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+   "description": "Allows user to easily set up domain using Entri",
+   "syncPubKeyDomain": "goentri.com",
+   "syncRedirectDomain": "api.goentri.com, goentri.com",
+   "variableDescription": "sendgridHost is sendgrid dynamic host, zendeskverification is the text verification value for zendesk, supportHost is the support host being added by fourthwall",
+   "hostRequired": true,
+   "sharedProviderName": true,
+   "records": [
+      {
+         "type": "CNAME",
+         "host": "%sendgridHost%.@",
```
Comment (pawel-kow) [inline goentri.com.fourthwall.subdomain.json:17]: `.@` is not needed here (and likely may be confused by some providers)

File: goentri.com.fourthwall.subdomain.json [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+         "pointsTo": "u10250786.wl037.sendgrid.net.",
+         "ttl": 3600
+      },
+      {
+         "type": "CNAME",
+         "host": "s1._domainkey.%supportHost%",
```
Comment (pawel-kow) [inline goentri.com.fourthwall.subdomain.json:23]: remove `.%supportHost%`

File: goentri.com.fourthwall.subdomain.json [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+         "pointsTo": "s1.domainkey.u10250786.wl037.sendgrid.net.",
+         "ttl": 3600
+      },
+      {
+         "type": "CNAME",
+         "host": "s2._domainkey.%supportHost%",
```
Comment (pawel-kow) [inline goentri.com.fourthwall.subdomain.json:29]: remove `.%supportHost%`

File: goentri.com.fourthwall.subdomain.json [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+         "pointsTo": "s2.domainkey.u10250786.wl037.sendgrid.net.",
+         "ttl": 3600
+      },
+      {
+         "type": "CNAME",
+         "host": "zendesk1._domainkey.%supportHost%",
```
Comment (pawel-kow) [inline goentri.com.fourthwall.subdomain.json:35]: remove `.%supportHost%`

File: goentri.com.fourthwall.subdomain.json [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+         "pointsTo": "zendesk1._domainkey.zendesk.com.",
+         "ttl": 3600
+      },
+      {
+         "type": "CNAME",
+         "host": "zendesk2._domainkey.%supportHost%",
```
Comment (pawel-kow) [inline goentri.com.fourthwall.subdomain.json:41]: remove `.%supportHost%`

File: goentri.com.fourthwall.subdomain.json [hunk 6]
```diff
@@ -0,0 +1,101 @@
...
+         "pointsTo": "zendesk2._domainkey.zendesk.com.",
+         "ttl": 3600
+      },
+      {
+         "type": "CNAME",
+         "host": "zendesk1.%supportHost%",
```
Comment (pawel-kow) [inline goentri.com.fourthwall.subdomain.json:47]: remove `.%supportHost%`

File: goentri.com.fourthwall.subdomain.json [hunk 7]
```diff
@@ -0,0 +1,101 @@
...
+         "pointsTo": "mail1.zendesk.com.",
+         "ttl": 3600
+      },
+      {
+         "type": "CNAME",
+         "host": "zendesk2.%supportHost%",
```
Comment (pawel-kow) [inline goentri.com.fourthwall.subdomain.json:53]: remove `.%supportHost%`

File: goentri.com.fourthwall.subdomain.json [hunk 8]
```diff
@@ -0,0 +1,101 @@
...
+         "pointsTo": "mail2.zendesk.com.",
+         "ttl": 3600
+      },
+      {
+         "type": "CNAME",
+         "host": "zendesk3.%supportHost%",
```
Comment (pawel-kow) [inline goentri.com.fourthwall.subdomain.json:59]: remove `.%supportHost%`

File: goentri.com.fourthwall.subdomain.json [hunk 9]
```diff
@@ -0,0 +1,101 @@
...
+         "pointsTo": "mail3.zendesk.com.",
+         "ttl": 3600
+      },
+      {
+         "type": "CNAME",
+         "host": "zendesk4.%supportHost%",
```
Comment (pawel-kow) [inline goentri.com.fourthwall.subdomain.json:65]: remove `.%supportHost%`

File: goentri.com.fourthwall.subdomain.json [hunk 10]
```diff
@@ -0,0 +1,101 @@
...
+         "pointsTo": "mail4.zendesk.com.",
+         "ttl": 3600
+      },
+      {
+         "type": "TXT",
+         "host": "zendeskverification.%supportHost%",
```
Comment (pawel-kow) [inline goentri.com.fourthwall.subdomain.json:71]: remove `.%supportHost%`

File: goentri.com.fourthwall.subdomain.json [hunk 11]
```diff
@@ -0,0 +1,101 @@
...
+         "data": "%zendeskverification%",
+         "ttl": 3600
+      },
+      {
+         "type": "TXT",
+         "host": "_dmarc.%supportHost%",
```
Comment (pawel-kow) [inline goentri.com.fourthwall.subdomain.json:77]: remove `.%supportHost%`

File: goentri.com.fourthwall.subdomain.json [hunk 12]
```diff
@@ -0,0 +1,101 @@
...
+         "data": "v=DMARC1; p=reject; pct=100; rua=mailto:dmarc@fourthwall.com",
+         "ttl": 3600
+      },
+      {
+         "type": "SPFM",
+         "host": "%supportHost%",
```
Comment (pawel-kow) [inline goentri.com.fourthwall.subdomain.json:83]: replace `%supportHost%` with `@`

File: goentri.com.fourthwall.subdomain.json [hunk 13]
```diff
@@ -0,0 +1,101 @@
...
+         "spfRules": "include:_spf.google.com include:mail.zendesk.com include:spf.improvmx.com include:sendgrid.net"
+      },
+      {
+         "type": "MX",
+         "host": "%supportHost%",
```
Comment (pawel-kow) [inline goentri.com.fourthwall.subdomain.json:88]: replace `%supportHost%` with `@`

File: goentri.com.fourthwall.subdomain.json [hunk 14]
```diff
@@ -0,0 +1,101 @@
...
+         "pointsTo": "mx1.improvmx.com.",
+         "priority": 10,
+         "ttl": 3600
+      },
+      {
+         "type": "MX",
+         "host": "%supportHost%",
```
Comment (pawel-kow) [inline goentri.com.fourthwall.subdomain.json:95]: replace `%supportHost%` with `@`

**Review by pawel-kow — CHANGES_REQUESTED** [review]

**Comment by beholders-eye** [comment]
Retrying it again, in a simpler way.

**Pull request closed without merging**